.github/workflows/vale.yml

```diff
@@ -0,0 +1,45 @@
+name: vale
+
+# Only run tests for main branch or if the PR has relevant changes
+on: 
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - 'docs/**'
+      - '**/*.md'
+
+concurrency:
+  group: vale-test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  vale:
+    name: vale
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
+          repository: splunk/vale-splunk-style-guide
+          path: vale-splunk-style-guide
+
+      - run: |
+          cat <<EOF > .vale.ini
```

Comment:
Is this Vale ini in conflict with the StylesPath below?

Reply:
Sorry, I don't follow. What are you referring to?

Reply:
Not sure how the setup works for a pipeline/action, but, essentially, the `.vale.ini` file already contains the path to styles. The path in the file must match whatever path in the local filesystem. Wondering if that's the cause for the pipeline not to work.

Reply:
I can try entering a nonsensical path and see if it gets picked up.